README.rst

```diff
@@ -0,0 +1,21 @@
+projects-edx-platform-extensions
+================================
+
+projects-edx-platform-extensions (``projects``) is a Django application responsible for managing the concept of Projects in the Open edX platform. Projects are an intersection of Courses, CourseContent, and Workgroups.
```

Comment:
@msaqib52 We should mention in the readme that this projects app had dependency on organizations app so organizations app should be installed already to install projects app.